udata/models/__init__.py

```diff
@@ -180,8 +180,10 @@ def init_app(app):
     for plugin in app.config['PLUGINS']:
         name = 'udata.ext.{0}.models'.format(plugin)
         try:
-            __import__(name)
-        except ImportError:
-            pass
-        except Exception as e:
+            module_obj = importlib.import_module(name)
+            for item in getattr(module_obj, '__all__'):
+                globals()[item] = getattr(module_obj, item)
```

Comment:
That's the only way I found to be able to import models from gouvfr via `udata.models` but I'm open to better solutions because it's ugly. To test by yourself: retrieve both udata and udata-gouvfr branches + generate and import new geozones, mmh I should write that somewhere else.